pykafka/balancedconsumer.py

```diff
@@ -553,6 +553,8 @@ def consumer_timed_out():
             try:
                 message = self._consumer.consume(block=block)
             except ConsumerStoppedException:
+                if hasattr(self, '_running') and not self._running:
```

Comment:
`_running` is set on `self` when start is called. If, for some reason, somebody calls consume before start I thought it will not be ok to see an AttributeError. The other solution is to set `self._running` to false in `__init__`. Do you think it's ok?